xave-finance/halodao-admin-scripts
# Feat: Reading and CSV Generation only

## Description of Changes
- Reading from firebase and generate CSV only
- There's a slight difference in rewards from the contract on other users, i added the firebase version and the contract version as well

[Link to Jira Ticket](https://halodao.atlassian.net/browse/HDF-270)

## How To Test
- get env on 1pass
- run `yarn run call:verify-address` and `npm run call:generate-airdroplist`


Note:
- i just merged to master directly since this is just something internal
- i just added all the contracts for reference but i can revise this and make them interfaces in the future

### Developer Checklist:


### Reviewers Checklist:
